### PR TITLE
feat: Krystal walletconnect

### PR DESCRIPTION
**Pull request by namgold**

### CODE REVIEW

File: src/hooks/web3/useActivationWallet.ts [hunk 1]
```diff
@@ -21,20 +21,26 @@ export const useActivationWallet: () => {
       const wallet = (SUPPORTED_WALLETS as any)[walletKey]
       if (!isEVMWallet(wallet)) return
       try {
-        console.info('Activate EVM start', { wallet, isEagerly })
-        // localStorage.removeItem(LOCALSTORAGE_LAST_WALLETKEY_EVM)
+        console.info('Activate EVM start', { walletKey, isEagerly })
         if (isEagerly) {
           if (wallet.connector.connectEagerly) {
             await wallet.connector.connectEagerly()
+            console.info('Activate EVM success with .connectEagerly()', {
```
Comment (nguyenhoaidanh) [inline src/hooks/web3/useActivationWallet.ts:28]: rm or keep it

Comment (namgold) [reply src/hooks/web3/useActivationWallet.ts:28]: we should keep it.

File: .github/workflows/pr.yaml [hunk 2]
```diff
@@ -114,7 +114,7 @@ jobs:
           VITE_TAG: ${{ needs.prepare.outputs.image_tag }}
           CURRENT_BRANCH: ${{ needs.prepare.outputs.current_branch }}
           NODE_OPTIONS: '--max_old_space_size=4096'
-        run: yarn build-adpr
+        run: yarn build-prod
```
Comment (nguyenhoaidanh) [inline .github/workflows/pr.yaml:117]: revert

File: src/constants/wallets.ts [hunk 3]
```diff
@@ -105,7 +107,12 @@ const detectRabby = (): WalletReadyState => {
 
 const detectKrystal = (): WalletReadyState => {
   if (getIsKrystalWallet()) return WalletReadyState.Installed
-  return WalletReadyState.NotDetected
+  return WalletReadyState.Unsupported
+}
+
+const detectKrystalWC = (): WalletReadyState => {
```
Comment (nguyenhoaidanh) [inline src/constants/wallets.ts:113]: detectKrystalWC vs detectKrystal: no difference

Comment (namgold) [reply src/constants/wallets.ts:113]: renamed for easier to understand

**Review by nguyenhoaidanh — DISMISSED** [review]

**Review by nguyenhoaidanh — APPROVED** [review]

**Review by nguyenhoaidanh — APPROVED** [review]

**Pull request merged**